data-raw/qc-markdowns/adult-upstream-passage-monitoring/deer-creek/deer-creek-upstream-passage-monitoring.Rmd

```diff
@@ -39,63 +39,413 @@ Some missing data for variable flow
+                                  sheet = "DEER-North-21-22",
+                                  skip = 5) |> 
+  mutate(ladder = "north") |>
+  glimpse()
+
+deer_2021_raw_south <- readxl::read_xlsx(paste0(save_to_prefix, "DCVS both 2021-22 Spring FINAL 3-3-23.xlsx"), 
+                                         sheet = "DEER-South-21-22",
+                                  
+                                  skip = 5) |> 
+  mutate(ladder = "south") |> 
+  glimpse()
+
+deer_2021 <- bind_rows(deer_2021_raw_north,
+                       deer_2021_raw_south) |> 
+  janitor::clean_names() |>
+  mutate(across(up_salmon:dn_jacks, as.numeric),
+         start_time = format(start_time, "%H:%M:%S"),
+         viewing_adjust = as.character(viewing_adjust)) |> 
+  select(date, start_time, up_salmon, dn_salmon, 
+         up_jacks, dn_jacks, viewing_adjust, ladder) |> 
+  pivot_longer(up_salmon:dn_jacks,
+               names_to = "count_type", 
+               values_to = "count") |> 
+  mutate(passage_direction = ifelse(count_type == "up_salmon", "up", "down"),
+         jack = ifelse(str_detect(count_type, "jacks"), TRUE, FALSE)) |> 
+  glimpse()
+```
+### 2022-2023
+- `north` and `south` ladders in different tabs
+
+```{r, message=FALSE, warning=FALSE}
+deer_2022_raw_north <- readxl::read_xlsx(paste0(save_to_prefix, "DCVS both 2022-2023 as of 10-16-23.xlsx"), 
+                                  sheet = "DEER-North-22-23",
+                                  skip = 5) |> 
+  mutate(ladder = "north") |>
+  glimpse()
+
+deer_2022_raw_south <- readxl::read_xlsx(paste0(save_to_prefix, "DCVS both 2022-2023 as of 10-16-23.xlsx"), 
+                                         sheet = "DEER-South-22-23",
+                                  
+                                  skip = 5) |> 
+  mutate(ladder = "south") |> 
+  glimpse()
+
+deer_2022 <- bind_rows(deer_2022_raw_north,
+                       deer_2022_raw_south) |> 
+  janitor::clean_names() |>
+  mutate(across(up_salmon:dn_jacks, as.numeric),
+         start_time = format(start_time, "%H:%M:%S"),
+         viewing_adjust = as.character(viewing_adjust)) |> 
+  select(date, start_time, up_salmon, dn_salmon, 
+         up_jacks, dn_jacks, viewing_adjust, ladder) |> 
+  pivot_longer(up_salmon:dn_jacks,
+               names_to = "count_type", 
+               values_to = "count") |> 
+  mutate(passage_direction = ifelse(count_type == "up_salmon", "up", "down"),
+         jack = ifelse(str_detect(count_type, "jacks"), TRUE, FALSE)) |> 
+  glimpse()
+```
+
+
+## Data transformations
+
+```{r, message=FALSE, warning=FALSE}
+all_years_raw <- bind_rows(deer_2013, deer_2014, deer_2015, deer_2016,
+                           deer_2017, deer_2018, deer_2019, deer_2020,
+                           deer_2021, deer_2022) |> 
+  glimpse()
+
+all_deer_data <- all_years_raw |> 
+  select(-count_type) |> 
+  # pulled from excel tabs
+  mutate(viewing_adjust = case_when(is.na(viewing_adjust) | viewing_adjust == 0 ~ "VAKI",
```

Comment:
how do you feel about all lower case? i know VAKI and DIDSON are abbrev but i like consistency in lower case.